pywinauto/timings.py

```diff
@@ -354,38 +414,9 @@ def WaitUntilPasses(
       except TimeoutError as e:
          print("timed out")
          raise e.
-     
-    """
     
-    start = time.time()
+    """    
+    return always_wait_until_passes(timeout, retry_interval, exceptions)(func)(*args)

```

Comment:
A plain function uses a decorator, not vice versa. Don't you find this a little bit strange?